pages/workflow.md

```diff
@@ -1,54 +1,107 @@
+{% include nav.html %} {% include css.html %}
+
 # Workflow
+	
+The information used for the visualization was gathered by using 
+the data provided by **www.slavevoyages.org** and after a bit of clean up that allowed the generated 
+data to work with other programs the workflow became more manageable. 
+
+Below are the steps taken that will demonstrate how the information generated
+by **slavevoyages.org** database was used in the site **Palladio** and the results the 
+information visually generated. Farther down the steps for how **Tableau** 
+will also be demonstrated. 
+
+***
+
+![image]({{ site.baseurl}}/assets/images/1. Prep for palladio File demo.png)
```

Comment:
Great documentation, but unreadable, since the images are shrunk so small. You might consider making the pictures clickable (linked), so the reader can see them full-sized. So would do it like so...